DegenPrimer/iPCR_Interface.py

```diff
@@ -41,9 +41,9 @@ def __init__(self, abort_event, job_id, *args, **kwargs):
         ReporterInterface.__init__(self)
         PCR_Simulation_Interface.__init__(self, abort_event, *args, **kwargs)
         self._job_id              = job_id
-        self._PCR_report_filename = '%s-%s-report.txt' % (self._job_id, 
+        self._PCR_report_filename = '%s-%s-report.txt' % (self._job_id[-20:],
                                                           self._PCR_report_suffix)
-        self._PCR_products_filename = '%s-%s-products.txt' % (self._job_id, 
+        self._PCR_products_filename = '%s-%s-products.txt' % (self._job_id[-20:],
```

Comment:
Same as above. We need the `--job-id` option to handle too long autogenerated job ids.